modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -76,4 +76,38 @@ definition {
 		}
 	}
 
+	@description = "LPS-185089 Verifies that the focus is not lost after closing the 'Edit Event' modal."
+	@priority = 3
+	test FocusIsNotLostAfterClosingTheModal {
+		SetWindowSize(value1 = "1920,1540");
+
+		task ("And a new event is added") {
+			CalendarNavigator.gotoAddEvent();
+		}
+
+		task ("When the user tries to edit the event") {
+			CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+		}
+
+		task ("And clicks any 'Select' button on 'CATEGORIZATION'") {
+			Click(locator1 = "Button#SELECT_CATEGORIES");
+		}
+
+		task ("When the modal is closed by the keyboard") {
+			KeyPress(
+				key_title = "Select Topic",
+				locator1 = "IFrame#MODAL_ANY",
+				value1 = "\TAB");
+
+			KeyPress(
+				key_modalTitle = "Select Topic",
+				locator1 = "Button#CLOSE_MODAL",
+				value1 = "\ENTER");
```

Comment:
Not sure if it is worth creating a macro for it.

Reply:
Is the tab key necessary for this? Can we get by with just the enter key?

Reply:
You're right... I can straight to the path I want to and press enter instead of pressing the tab key to reach it.